src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -774,4 +728,81 @@ private function getSerializerContext(string $operationType, bool $denormalizati
 
         return $resourceMetadata->getItemOperationAttribute($operationName, $contextKey, null, true);
     }
+
+    /**
+     * @param ResourceMetadata $resourceMetadata
+     * @param string           $operationName
+     * @param \ArrayObject     $pathOperation
+     */
+    private function addPaginationParameters(ResourceMetadata $resourceMetadata, string $operationName, \ArrayObject $pathOperation)
+    {
+        if ($this->paginationEnabled && $resourceMetadata->getCollectionOperationAttribute($operationName, 'pagination_enabled', true, true)) {
+            $pathOperation['parameters'][] = $this->getPaginationParameters();
+
+            if ($resourceMetadata->getCollectionOperationAttribute($operationName, 'pagination_client_items_per_page', $this->clientItemsPerPage, true)) {
+                $pathOperation['parameters'][] = $this->getItemsParPageParameters();
+            }
+        }
+    }
+
+    /**
+     * @param array            $subresourceOperation
```

Comment:
Same here, you can drop all the `@param` and `@return`. But keep the `@throw` :)